godel_robots/abb/godel_irb2400/godel_irb2400_support/launch/irb2400_blending_client.launch

```diff
@@ -9,16 +9,22 @@
   <arg name="robot_model_plugin" default="abb_irb2400_descartes/AbbIrb2400RobotModel"/>
 
   <group if="$(arg sim_sensor)" ns="$(arg namespace)">
```

Comment:
You could create an all encompassing "**<group ns="$(arg namespace)">**" scope that contains all of the launch file elements (nodes, includes, etc.) that need to be placed under the same namespace.